scalding-hadoop-test/src/test/scala/com/twitter/scalding/platform/PlatformTest.scala

```diff
@@ -701,6 +717,13 @@ class PlatformTest extends WordSpec with Matchers with HadoopSharedPlatformTest
     }
   }
 
+  "An EmptyData source" should {
+    "read from empty source and write to output without errors" in {
+      HadoopPlatformJobTest(new EmptyDataJob(_), cluster)
+        .run()
+    }
+  }
+
```

Comment:
Yeah so that's cause we're currently not calling validateTaps in the HadoopJobTest. Updated the code to mixin SuccessFileSource as well.